x-pack/plugins/watcher/public/sections/watch_edit/components/threshold_watch_edit_component.tsx

```diff
@@ -461,11 +484,25 @@ const ThresholdWatchEditUi = ({
               button={
                 <EuiExpression
                   description={`${watch.groupBy === 'all' ? 'over' : 'grouped over'}`}
-                  value={groupByTypes[watch.groupBy].text}
-                  isActive={groupByPopoverOpen}
+                  value={`${groupByTypes[watch.groupBy].text} ${
+                    watch.groupBy === 'all'
+                      ? ''
+                      : (watch.termSize || '') +
```

Comment:
do you think this would be easier to read using template literals?

Reply:
yeah this is gross, will see if I can clean it up